firmware/arduino/chessboard/uart.ino

```diff
@@ -3,6 +3,10 @@ char buffer[6];
 int byteNum = -1; // -1 indicates that the start code hasn't been received
 char currentState = '0';
 char errorCode;
+volatile unsigned long previous_activation_time = 0;
+const unsigned long DEBOUNCE_TIME = 100; // Milliseconds
+bool movementFlag = false;
+volatile bool transmitFlag = false;
 
 enum ArduinoState
```

Comment:
@nashirj I'm not sure about this, but wouldn't it be better to store these in preprocessors since they wouldn't take up memory?

Reply:
I think the variables are fine as is, but the const can be changed to a define